internal/services/cosmos/cosmosdb_account_resource.go

```diff
@@ -767,6 +808,41 @@ func resourceCosmosDbAccountUpdate(d *pluginsdk.ResourceData, meta interface{})
 		return fmt.Errorf("making Read request on %s: %s", id, err)
 	}
 
+	oldCapabilities := make([]documentdb.Capability, 0)
+	oldCapabilitiesMap := make(map[string]interface{})
+	for _, v := range *resp.Capabilities {
+		c := documentdb.Capability{
+			Name: v.Name,
+		}
+		oldCapabilities = append(oldCapabilities, c)
+		oldCapabilitiesMap[strings.ToLower(*c.Name)] = *c.Name
+	}
```

Comment:
by directly using `read.Properties.Capabilities` below, we can remove this:

```suggestion
```

Reply:
Fixed.